laravel/app/Console/Commands/InitiateEval.php

```diff
@@ -105,152 +106,114 @@ public function initResidentEvalAttendingPOST($evaluatorID, $evaluateeID)
-                                $evaluateeService
-                        );
-                    }
-                    try {
-                        self::initResidentEvalAttendingPOST($evaluateeID, $evaluatorID);
-                        $evalSent++;
-                    } catch (\Exception $e) {
-                        Log::debug(
-                            "Error on Resident Eval Attending. Eval ID" .
-                                $evalID[$i] .
-                                " Attending ID: " .
-                                $evaluatorID .
-                                "  Resident Name " .
-                                $evaluateeName .
-                                " Resident ID " .
-                                $evaluateeID
-                        );
-                    }
+        }
+        return $evalsSent;
+    }
 
-                    // if used REMODEL, fire off additional evaluation
-                    $hasOptions = DB::table("option")
-                        ->where("resident", $evaluateeID)
-                        ->where("date", $yesterday)
-                        ->value("date");
-                    if ($hasOptions != null) {
-                        try {
-                            $additionalFormType = "REMODEL feedback";
-                            $additionalService = EvaluationForms::where("form_type", $additionalFormType)->value(
-                                "medhub_form_id"
-                            );
-                            self::initAttendingEvalResidentPOST($evaluatorID, $evaluateeID, $additionalService);
-                            $evalSent++;
-                        } catch (\Exception $e) {
-                            Log::debug(
-                                "Error on Attending Eval Resident. Eval ID" .
-                                    $evalID[$i] .
-                                    " Resident Name: " .
-                                    $evaluateeName .
-                                    " Resident ID " .
-                                    $evaluateeID .
-                                    " Attending ID " .
-                                    $evaluatorID .
-                                    " Service ID" .
-                                    $evaluateeService
-                            );
-                        }
-                    }
+    public function initiateEvaluations()
+    {
+        $date = date("Y-m-d");
+        $date = strtotime("-1 day", strtotime($date));
+        $date = date("Y-m-d", $date);
+        Log::info("initiate evaluations for " . $date . " evaluation data");
+
+        $alwaysEvalServices = ["1", "2", "21", "23"]; // hardcoded for now - needs to be changed when db updated
+        $totalEvalsSent = 0;
+        $time_difference = DB::table("variables")->where("name", "time_before_attending_evaluates_resident")->value("value");
+        $time_difference = (int)$time_difference;
```

Comment:
I'm wanted to double check this, and I was wrong! The value column is actually a `varchar(255)` (presumably b/c other future variables could need a string value).

![image](https://user-images.githubusercontent.com/9152032/114225825-cd860080-9940-11eb-814a-8e77693d7fbe.png)

The `(int)` cast should stay, but we don't need 2 lines to do it.

Alternatively, it may be better to do something like [this](https://stackoverflow.com/a/13502388/13173164) using `intval`.